Mathlib/Data/Complex/Module.lean

```diff
@@ -43,7 +44,7 @@ element of a `StarModule` over `ℂ`.
   `ComplexStarModule`.
 -/
 
-
+assert_not_exists NNReal
```

Comment:
A vague justification for this in a comment would be nice; otherwise it's harder to review a PR in future that removes it.

Reply:
Sorry, I'm not sure what justification would look like beyond "that notion is not needed to define this notion", and in any case the way to review a PR that removes it is to say "no". :-)